alarm/models.py

```diff
@@ -14,26 +14,44 @@ class Phone(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     number = PhoneNumberField(blank=True, unique=True, region='UA')
     enabled = models.BooleanField(default=False)
+    message = models.TextField(default=None)
 
     def __str__(self):
         return str(self.number)
 
+    def refresh_message(self, message):
+        self.message = message
+        self.save()
+
+    def clear_old_message(self):
```

Comment:
That is really good encapsulation, like for it.

Reply:
++